crowbar_framework/app/models/service_object.rb

```diff
@@ -1647,6 +1681,11 @@ def run_remote_chef_client(node, command, logfile_name)
 
   THREAD_POOL_SIZE = 20
 
+  # Is the upgrade of (some) compute nodes postponed
+  def upgrade_postponed?
+    Dir.glob("/var/lib/crowbar/upgrade/*-upgrade-compute-nodes-postponed").any?
```

Comment:
should this point to the exact versioning for the upgrade (i.e. "7-8") or is that file not versioned yet?